### DISCUSSION TITLE
Fault showing on Forward Sensor via Grid

### DISCUSSION THREAD

**Discussion by kl3ss** [Security-Onion-Solutions/securityonion#5934] [Unsupported Versions]
Hey all,

Was wondering how one would go about diagnosing a Fault on the Sensor Grid? All my forward nodes on the grid initially reported green good but now display a red fault. There is no information to say what the fault is? The only thing I notice is that Zeek is missing on my Forward Nodes.

Cheers

KL3SS

**Comment by dougburks** [comment]
Do you have Internet access or this an airgap deployment?

Did you update using `soup` and were there any errors? You can try using `so-docker-refresh`:
https://docs.securityonion.net/en/2.3/soup.html#errors

**Reply by kl3ss** [reply]
Hey Doug, 

Thanks for the quick reply, This is an airgap deployment. Should I follow the instructions of updating an air-gaped deployment and see if that fixes the issue?

Cheers
KL3SS

**Reply by kl3ss** [reply]
Also Doug,

Can we run the soup -y -f /path/to/iso/securityonion.iso on the forward nodes themselves or is that not recommended?

Cheers

KL3SS

**Reply by dougburks** [reply]
> The only thing I notice is that Zeek is missing on my Forward Nodes.

When you ran Setup and it asked about metadata, did you choose Zeek or Suricata?

> This is an airgap deployment. Should I follow the instructions of updating an air-gaped deployment and see if that fixes the issue?

My question was really about if the system was showing green good and then you ran soup and that's when things changed as sometimes Internet connected deployments have issues downloading the new container images. Since you're running an airgap deployment, that shouldn't be an issue for you.

Does the deployment appear to be functioning normally other than the fault itself?

Are there any network firewalls that may be restricting traffic between the forward nodes and the manager?

> Can we run the soup -y -f /path/to/iso/securityonion.iso on the forward nodes themselves or is that not recommended?

You only need to run soup on the manager and that will update the entire deployment automatically. However, you probably want to make sure that your deployment is running properly before attempting soup.

**Reply by kl3ss** [reply]
Hey Doug,

During Setup we did choose Suricata for our Metadata.

We have not run an update / soup yet. This is due to being in an air-gaped environment.

Forward Sensors were initially Green but now report a fault.

The environment does have firewalls.

The deployment does appear to be functioning normally.

**Reply by dougburks** [reply]
> During Setup we did choose Suricata for our Metadata.

If you chose Suricata for metadata, then Zeek shouldn't be running at all.

> The environment does have firewalls.

Have you verified that those firewalls allow all traffic from forward sensors to manager as shown at https://docs.securityonion.net/en/2.3/firewall.html#node-communication?
